test-cases/nemesis/longevity-5gb-1h-BlockNetworkMonkey.yaml

```diff
@@ -19,11 +19,12 @@ instance_type_loader: 'c5.large'
 azure_instance_type_loader: 'Standard_F2s_v2'
 
 extra_network_interface: true
+ip_ssh_connections: 'public'  # need to use public IPs because extra NIC is in the same subnet as default one.
 
 nemesis_class_name: 'BlockNetworkMonkey'
 nemesis_interval: 3
 nemesis_filter_seeds: false
-nemesis_during_prepare: true
+nemesis_during_prepare: false
```

Comment:
I don't think we should set it to false.

Reply:
undone